File: Model/src/main/java/domain/validators/TicketValidator.java
package domain.validators;

import domain.Ticket;

import java.io.Serializable;

public class TicketValidator implements Validator<Ticket>, Serializable {
    @Override
    public void validate(Ticket entity) throws ValidationException {
        if(entity.getClient().isEmpty())
            throw new ValidationException("Numele clientului nu poate fi vid!");
        if(entity.getFestival()==null)
            throw new ValidationException("Festivalul pentru care se vinde biletul nu poate fi null!");
    }
}

File: model/Validators/TicketValidator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LabMPP.domain.Validators
{
    public class TicketValidator : Validator<Ticket>
    {
        public void validate(Ticket entity)
        {
            if (entity.client=="")
                throw new ValidationException("Numele clientului nu poate fi vid!");
            if (entity.festival == null)
                throw new ValidationException("Festivalul pentru care se vinde biletul nu poate fi null!");
        }
    }
}
